vendor/modules.txt

```diff
@@ -990,7 +990,7 @@ k8s.io/kubernetes/test/e2e/storage/podlogs
 k8s.io/kubernetes/test/e2e/storage/utils
 k8s.io/kubernetes/test/utils
 k8s.io/kubernetes/test/utils/image
-# k8s.io/mount-utils v0.22.0 => k8s.io/mount-utils v0.22.0
+# k8s.io/mount-utils v0.22.1 => k8s.io/mount-utils v0.22.0
```

Comment:
```suggestion
# k8s.io/mount-utils v0.22.1 => k8s.io/mount-utils v0.22.1
```

It should've been this. 
@nixpanic ^^

Reply:
good catch! maybe we can remove the `k8s.io` packages from `replace`?

Reply:
@nixpanic if we remove it from `replace` section, I expect some failures in compiling.

Reply:
yes, I tried it out. It seems `k8s.io/kubernetes` depends on this (and many other packages) without specifying a version as the sub-packages are local to kubernetes (in `./staging/src/k8s.io/...`). This package, and others need to be ignored by dependabot (#2461), and updates to `k8s.io/kubernetes` need to be adjusted to update the sub-packages as well (#2455).